src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp

```diff
@@ -542,7 +542,7 @@ namespace
                         {
                             Color3f variation;
                             m_variation_aov_tile->get_pixel(pt.x, pt.y, variation);
-                            variation[0] += pb.m_block_error;
+                            variation[0] += pb.m_block_error / m_params.m_noise_threshold;
```

Comment:
That's actually a pretty nice solution as it doesn't require injecting the noise threshold into the AOV.

You can precompute `1.0f / m_params.m_noise_threshold`.

Reply:
Thanks @dictoon . I made the change.
I realized that we did not need to inject the noise threshold into the AOV since the variation was computed in the Adaptive Tile renderer.